### PR TITLE
feat: AnnLite indexer on ci + secure + only AnnLite indexer + replace CaS CLIP executor

### LINKED ISSUE #488: feat: replace CaS CLIP executor with smaller CLIP model on external executor

### PR DESCRIPTION
**Pull request by joschkabraun**
- [x] closes https://github.com/jina-ai/now/issues/458 by putting building of `NOWAnnLiteIndexer` on CI
- [x] closes https://github.com/jina-ai/now.private/issues/124 by letting `NOWAnnLiteIndexer` extend `NOWAuthExecutor`
- [x] closes https://github.com/jina-ai/now/issues/485 by remove usage of DocArrayIndexer
- [x] closes #488 by replacing CaS CLIP executor with original smaller model as quality is worse for larger model

### CODE REVIEW

File: now_executors/NOWAnnLiteIndexer/executor.py [hunk 1]
```diff
@@ -3,11 +3,13 @@
 from typing import Dict, List, Optional
 
 import annlite
-from jina import Document, DocumentArray, Executor, requests
+from jina import Document, DocumentArray
 from jina.logging.logger import JinaLogger
+from now_executors import NOWAuthExecutor as Executor
```
Comment (florian-hoenicke) [inline now_executors/NOWAnnLiteIndexer/executor.py:8]: How does it know about the this path?
shouldn't it be `now.now_executors....`?

Comment (joschkabraun) [reply now_executors/NOWAnnLiteIndexer/executor.py:8]: > How does it know about the this path?
> shouldn't it be `now.now_executors....`?

in `now_common/__init__.py` it's imported

Comment (joschkabraun) [reply now_executors/NOWAnnLiteIndexer/executor.py:8]: also e.g. `now_common/...` works as import path

File: now/constants.py [hunk 2]
```diff
@@ -47,10 +48,12 @@ class Qualities(BetterEnum):
 )
 
 CLIP_USES = {
-    'local': ('CLIPTorchEncoder/latest', 'ViT-B-32::openai', 512),
-    'remote': ('CLIPTorchEncoder/latest-gpu', 'ViT-L-14-336::openai', 768),
+    'local': ('CLIPOnnxEncoder/latest', 'ViT-B-32::openai', 512),
+    'remote': ('CLIPOnnxEncoder/latest-gpu', 'ViT-B-32::openai', 512),
 }
 
+EXTERNAL_CLIP_HOST = 'encoderclip-bh-5f4efaff13.wolf.jina.ai'
```
Comment (florian-hoenicke) [inline now/constants.py:55]: I think next step will be to create a DNS entry in AWS or cloudflare and have the static address here. But for now we can leave it.

File: now_executors/NOWAnnLiteIndexer/executor.py (hunk 1 above)
Comment (florian-hoenicke) [reply now_executors/NOWAnnLiteIndexer/executor.py:8]: Nice that it works

**Comment by florian-hoenicke** [comment]
LGTM

**Review by florian-hoenicke — APPROVED** [review]

**Pull request merged**